module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -79,7 +89,7 @@ public function editAllowed($user)
     }
 
     /**
-     * Get an array of tags associated with this list.
+     * Get an array of resource tags associated with this list.
```

Comment:
Maybe it's worth renaming the method to match this comment. You could keep a deprecated getTags() method as an alias for getResourceTags() for backward compatibility. This might make other code more clear.